mlir/include/mlir/Dialect/MIOpen/Tuning/GridwiseGemmParams.h

```diff
@@ -415,6 +425,64 @@ class PopulateParamsBase {
     return mlir::success();
   }
 
+  mlir::LogicalResult calculateOutputDerivedParams(InitParams *params,
+                                                   int64_t blockSize,
+                                                   ConvolutionContext &ctx,
+                                                   DerivedOutParams &out) {
+    int64_t cVectorLength = 0;
+    ConvOpType op = ctx.getOpType();
+
+    obtainGemmCVecLen(ctx, cVectorLength);
+    int64_t dataPerThread =
+        (params->gemmMPerBlock * params->gemmNPerBlock) / blockSize;
+    if (!(dataPerThread > 0)) {
+      return failure();
+    }
+
+    // TODO: Allow vectorization group size of 2
+    int64_t vectorizationSize = 4;
+    if (ConvOpType::Conv2DBwdDataOpType == op) {
+      vectorizationSize = 1;
```

Comment:
comment might be neede to state we are disabling vector stores for backward data convolution.